### PR TITLE
From FAIR software to Software in the Research Landscape

### PR DESCRIPTION
**Pull request by JaroCamphuijsen**
Revise FAIR module to be more general about the role of software in science and research software landscape

- Rename module, folders, files and titles
- Take away focus on FAIR
- Add general overview of initiatives in the research landscape
- Make exercises more general

### CODE REVIEW

**Comment by JaroCamphuijsen** [comment]
@raar1 I'm curious for you opinion on these changes. I think it will need some more work, but would like your review already if you don't mind.

File: modules/softwarelandscape/slides.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+---
+
+<!-- .slide: data-state="title" -->
+
+# Software in the Scientific Landscape
+
+===
+
+<!-- .slide: data-state="standard center" -->
+## Some general observations:
```
Comment (raar1) [inline modules/softwarelandscape/slides.md:14]: I like the train of thought here but it is quite a lot to have on one slide

File: modules/softwarelandscape/exercise2.md [hunk 2]
```diff
@@ -4,44 +4,14 @@ type: exercise
...
 ---
 
-## Exercise: Mapping the principles
+## **Exercise:** Mapping the principles
 
-### The FAIR4RS Principles are:
+Choose one of the three examples of software below. They have been described with statements that can be mapped to the FAIR4RS Principles. With each statement in the software description, identify the principle(s) it maps to, and the (sub)facet(s) of the principle(s) it addresses. A single statement may map to multiple (sub)facets.
 
-#### F: Software, and its associated metadata, is easy for both humans and machines to find.
+![FAIR for Research Software principles](media/fair4rs_principles.png)
```
Comment (raar1) [inline modules/softwarelandscape/exercise2.md:11]: Is there a way to have the deleted text as alt text for this image? Just thinking about screen readers or something not being able to parse the image so easily. I do personally find the image easier and more pleasant to read, though.

File: modules/softwarelandscape/exercise1.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+title: Study & present
+type: exercise
+order: 2
+---
+
+## Exercise: Study and present (20 - 30 minutes)
```
Comment (raar1) [inline modules/softwarelandscape/exercise1.md:7]: This is quite a nice idea for an exercise as we might also get to hear views from participants who have interacted in these spaces as part of their job already.

File: modules/softwarelandscape/exercise3.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+Pick a piece of software you know of or pick one from the following examples:
+
+- [ESMValTool](https://research-software-directory.org/software/esmvaltool)
+- [LitStudy](https://research-software-directory.org/software/litstudy)
+- [Haddock](https://research-software-directory.org/software/haddock3)
+- [worcs](https://cjvanlissa.github.io/worcs/index.html)
+- [democracy-topic-modelling](https://research-software-directory.org/software/democracy-topic-modelling)
+
+
+One person (the researcher) takes some time to prepare to presents a piece of software, the other person reads up on the [reading materials](reading.md) from this module.
```
Comment (raar1) [inline modules/softwarelandscape/exercise3.md:17]:
```suggestion
One person (the researcher) takes some time to prepare to present a piece of software, while the other person reads up on the [reading materials](reading.md) from this module. 
```

?

File: modules/softwarelandscape/exercise3.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+- [Haddock](https://research-software-directory.org/software/haddock3)
+- [worcs](https://cjvanlissa.github.io/worcs/index.html)
+- [democracy-topic-modelling](https://research-software-directory.org/software/democracy-topic-modelling)
+
+
+One person (the researcher) takes some time to prepare to presents a piece of software, the other person reads up on the [reading materials](reading.md) from this module. 
+
+Then you play out one of the following two situations:
+
+1. A researcher comes to a supporter with their piece of software and asks for support on managing their piece of software.
```
Comment (raar1) [inline modules/softwarelandscape/exercise3.md:21]:
```suggestion
1. A researcher comes to a supporter with their piece of software and asks for support on managing it.
```

Maybe?

File: modules/softwarelandscape/exercise3.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+2. A research supporter approaches a researcher which is, according to the organization's records, contributer to the software.
```
Comment (raar1) [inline modules/softwarelandscape/exercise3.md:22]:
```suggestion
2. A research supporter approaches a researcher who is, according to the organization's records, a contributer to the software.
```

File: modules/softwarelandscape/exercise3.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
+---
+title: Role playing
+type: exercise
+order: 4
+---
+
+## Exercise: Role playing exercise (15 minutes)
```
Comment (raar1) [inline modules/softwarelandscape/exercise3.md:7]: Nice, I think these role playing exercises are the right way to go for training support staff

**Review by raar1 — CHANGES_REQUESTED** [review]
This is good work and I agree with the move away from only talking about FAIR. I have left a couple of comments and suggestions.

A couple of other points (although I'm sure you're aware since this is WIP):
* The learning objectives need to be updated I think (currently they're FAIR-only).
* Likewise, maybe add some links to the institutions/orgs mentioned "software landscape" to the Further Reading section?

I'm happy to review this again on Monday too :)

File: modules/softwarelandscape/exercise2.md (hunk 2 above)
Comment (JaroCamphuijsen) [reply modules/softwarelandscape/exercise2.md:11]: We will change this into a table, an issue was opened: https://github.com/esciencecenter-digital-skills/research-software-support/issues/124

File: modules/softwarelandscape/slides.md (hunk 1 above)
Comment (JaroCamphuijsen) [reply modules/softwarelandscape/slides.md]: This was addressed in https://github.com/esciencecenter-digital-skills/research-software-support/pull/121/commits/7aefb650908186ae75b13d7a1205cf9f9779adc2

**Pull request merged**